include/elliptics/packet.h

```diff
@@ -1000,6 +1000,10 @@ static inline void dnet_convert_iterator_request(struct dnet_iterator_request *r
 	dnet_convert_time(&r->time_end);
 }
 
+
+/* Common record flags */
+#define DNET_RECORD_FLAGS_UNCOMMITTED	(1<<0) /* The record was prepared but hasn't been committed */
```

Comment:
'uncommitted' is still a eblob-specific flag. We can try to map flags of the different backends to these new eblob-derived flags if needed, but I do not think this will be ever needed - for the last 6 years we had about 5-6 different backends and there were no external flags at all (and backends were removed eventually).

Having more eblob flags in iterator output and callback allows to check whether it has external header, if we will ever have structured objects stored, this flag will also help to check this.